spec/spec_helper.rb

```diff
@@ -123,6 +123,9 @@ class Query
 end
 
 I18n.config.enforce_available_locales = false
+if %w(yes true 1).include?((ENV['TEST_I18N_FALLBACKS'] || '').downcase)
+  include I18n::Backend::Fallbacks
```

Comment:
This includes into the global scope no? If you want `I18n.fallbacks` defined at this point this should just `require` the backend.